CMakeLists.txt

```diff
@@ -103,7 +88,13 @@ GENERATE_EXPORT_HEADER( AcesContainer
              STATIC_DEFINE AcesContainer_BUILT_AS_STATIC
 )
 
-install (TARGETS AcesContainer EXPORT AcesContainerTargets DESTINATION ${INSTALL_LIB_DIR})
+# Set the build version (VERSION) and the API version (SOVERSION)
+set_target_properties(AcesContainer
+	PROPERTIES
+	VERSION ${AcesContainer_VERSION}
+	SOVERSION ${AcesContainer_MAJOR_VERSION})
+
+install (TARGETS AcesContainer EXPORT AcesContainerTargets DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
On Windows, need to make sure binary artifacts (DLL) is installed into `bin` while import library in `lib`. This is done by defining RUNTIME, ARCHIVE, and LIBRARY destinations: https://cmake.org/cmake/help/latest/command/install.html#installing-targets

Reply:
I Can change that, but I won't be able to test.

As I undertstood, I need to have:
install (TARGETS AcesContainer EXPORT AcesContainerTargets
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
)

Reply:
Yes!

Reply:
OK, I have a flight tonight so after Easter I will try to add commit to fix theses issue. Hopefully  next week.